zhaquirks/sonoff/swv.py

```diff
@@ -0,0 +1,71 @@
+"""Sonoff SWV - Zigbee smart water valve."""
+
+import typing
+
+from zigpy.quirks import CustomCluster
+from zigpy.quirks.v2 import QuirkBuilder
+from zigpy.quirks.v2.homeassistant.binary_sensor import BinarySensorDeviceClass
+import zigpy.types as t
+from zigpy.zcl.foundation import BaseAttributeDefs, ZCLAttributeDef
+
+
+class ValveState(t.enum8):
+    """Water valve state."""
+
+    Water_Shortage = 1
+    Water_Leakage = 2
```

Comment:
To show all the possible values if querying the cluster directly, can you also add the Normal (0) and Water leakage and shortage (3) values?